Documents/QA/TestCases.md

```diff
@@ -0,0 +1,50 @@
+## This is the template of how the test cases will be documented for this project.
+
+
+| ID | Date  | Priority | Test case Scenario                                  | Test case Description     | Pre-conditions    | Test Steps                   | Expected Results                                                |
+| -- | --- | -------- |------------------------------------------------------------------------------------------ | ------ | --------------------------------- | --------- | ----------------------------------------------------------------------------------------------------------------- |
+| 01/001 | 01/01/1970 | HIGH | run the program, and see what's displayed | make sure the 'printf' function works correctly | have the most up-dated version of the product | 1. run the program 2. see the result | "Hello World" is displayed |
+
+**ID**<br>
+The ID of the test case. It will follow a certain pattern: 
+- The first numbers are the version of the tested product -> `01`/001 <br>
+The versions are:

```

Comment:
problem with printf, which does not exist in Verilog :)

Reply:
done